cms/serializers.py

```diff
@@ -27,19 +31,23 @@ def get_feature_image_src(self, instance):
 
         return feature_img_src or static(DEFAULT_COURSE_IMG_PATH)
 
+    @extend_schema_field(serializers.URLField)
     def get_page_url(self, instance):
         return instance.get_url()
 
+    @extend_schema_field(str)
     def get_description(self, instance):
         return bleach.clean(instance.description, tags=[], strip=True)
 
+    @extend_schema_field(str)
     def get_effort(self, instance):
```

Comment:
```suggestion
    def get_effort(self, instance) -> str | None:
```

typo annotations are good enough in simple cases, and this one should be nullable.

A downside of drf-spectacular is that we do need to manually provide some types, like here.

That said, we do a lot less manually than we would if we just wrote TS definitions for all the types in frontend files.